pom.xml

```diff
@@ -5,7 +5,7 @@
     <name>Signature API-client Java</name>
     <groupId>no.digipost.signature</groupId>
     <artifactId>signature-api-client-java</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0-STATUS-API-SNAPSHOT</version>
```

Comment:
Also: slightly annoying :D I agree with your comment regarding automation in https://github.com/digipost/signature-api-specification/pull/2#discussion_r42212904. I'll look into writing a script for this purpose.